app/uk/gov/hmrc/cdsimportsddsfrontend/domain/DocumentationType.scala

```diff
@@ -16,41 +16,57 @@
 
 package uk.gov.hmrc.cdsimportsddsfrontend.domain
 
+case class WriteOffType(quantity: Option[String])
+
+case class AdditionalDocumentType(
+  categoryCode: Option[String],
+  typeCode: Option[String],
+  id: Option[String],
+  lpco: Option[String],
+  name: Option[String],
+  effectiveDateTime: Option[String],
+  submitter: Option[String],
+  writeOff: Option[String]
+)
+
+case class AdditionalPaymentType(
+  additionalDocPaymentID: Option[String],
+  additionalDocPaymentCategory: Option[String],
+  additionalDocPaymentType: Option[String]
+)
+
 case class DocumentationType(
```

Comment:
The name shouldn't include the word `Type`

The CDS PDF name for section 2 is References so this class should be called References